apps/api/src/middleware/auth.js

```diff
@@ -0,0 +1,21 @@
+import passport from "passport";
+
+export const middlewares = {
+  ensureAuthenticated: (req, res, next) => {
```

Comment:
I think this is better

```suggestion
export const ensureAuthenticated = (req, res, next) => {
```

Reply:
So we avoid

```js
import { middlewares } from './middlewares';

app.use(middlewares. ensureAuthenticated)
```

instead this would be

```js
import { ensureAuthenticated } from './middlewares';

app.use(ensureAuthenticated)
```